src/ape_accounts/_cli.py

```diff
@@ -64,22 +64,36 @@ def _list(cli_ctx, show_all_plugins):
             click.echo()
 
 
-@cli.command(short_help="Create a new keyfile account with a random private key")
+@cli.command(short_help="Create a new keyfile account with a random mnemonic seed phrase")
+@click.option(
+    "--word-count",
+    "word_count",
```

Comment:
You don't actually need this, click will automatically convert the variable name to this